rails_translation_manager.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rails-i18n"
   spec.add_dependency "activesupport"
 
-  spec.add_development_dependency "bundler", "~> 1.7"
+  spec.add_development_dependency "bundler"
```

Comment:
This can be rebased out now that #14 is merged.

Reply:
Done!